src/Sulu/Component/Content/Exception/ResourceLocatorAlreadyExistsException.php

```diff
@@ -15,4 +15,47 @@
 
 class ResourceLocatorAlreadyExistsException extends Exception
 {
+    /**
+     * @var string
+     */
+    private $resourceLocator;
+
+    /**
+     * @var string
+     */
+    private $path;
+
+    public function __construct($resourceLocator, $path)
+    {
+        $this->resourceLocator = $resourceLocator;
+        $this->path = $path;
+        parent::__construct(
+            sprintf(
+                'Resource locator "%s" already exists. Route node at path "%s"',
```

Comment:
Can you provide a more helpful error message? Why does this error occur? What information does the reader of the exception need to recover?